packages/core/src/lazynodes.rs

```diff
@@ -23,7 +23,7 @@ use crate::{innerlude::VNode, ScopeState};
 ///
 ///
 /// ```rust, ignore
-/// LazyNodes::new(|f| f.element("div", [], [], [] None))
+/// LazyNodes::new(|f| f.render(rsx!(div {})).unwrap())
```

Comment:
If you want to use rsx, you shouldn't wrap it in LazyNodes::new(). rsx!(div{}) expands to something like LazyNodes::new(|f| ...create div):

```rust
::dioxus::core::LazyNodes::new(move |
        __cx: &::dioxus::core::ScopeState,
    | -> ::dioxus::core::VNode {
        static TEMPLATE: ::dioxus::core::Template = ::dioxus::core::Template {
            name: "src/main.rs:5:5:83",
            roots: &[
                ::dioxus::core::TemplateNode::Element {
                    tag: dioxus_elements::div::TAG_NAME,
                    namespace: dioxus_elements::div::NAME_SPACE,
                    attrs: &[],
                    children: &[],
                },
            ],
            node_paths: &[],
            attr_paths: &[],
        };
        ::dioxus::core::VNode {
            parent: None,
            key: None,
            template: std::cell::Cell::new(TEMPLATE),
            root_ids: dioxus::core::exports::bumpalo::collections::Vec::with_capacity_in(
                    1usize,
                    __cx.bump(),
                )
                .into(),
            dynamic_nodes: __cx.bump().alloc([]),
            dynamic_attrs: __cx.bump().alloc([]),
        }
    })
```

That makes the expanded version of this code something like:

```rust
LazyNodes::new(|f| **resolve lazy nodes to the elements they create** LazyNodes::new(|f| **actually create the elements**))
```

We could show the expanded version of `rsx!{div{}}` instead and link to https://dioxuslabs.com/learn/0.4/contributing/walkthrough_readme#the-rsx-macro.

Reply:
Alright, changed it to a closure that makes an empty div element